### DISCUSSION TITLE
Mac computer, cannot open folders

### DISCUSSION THREAD

**Discussion by vtoepel** [phcode-dev/phoenix#676] [General]
When using Phoenix in Chrome on a Mac, it will not let me open folders.

When I click open folder, nothing happens.

**Comment by abose** [comment]
Hi @vtoepel Thanks for the report. Is this a recent issue?
Try reloading Phoenix and see if the issue still persists.
If the issue still exists, Please provide the following details:

1. Mac OS version and is it M1/intel mac.
2. chrome version
1. open developer tools and paste any logs/errors in the console here.
